src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,90 @@
+package wolox.training.models;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import wolox.training.exceptions.BookAlreadyOwnedException;
+import wolox.training.exceptions.BookNeverOwnedException;
+
+@Entity(name = "USERS")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+    @Column(nullable = false)
+    private String username;
+    @Column(nullable = false)
+    private String name;
+    @Column(nullable = false)
+    private LocalDate birthdate;
+
+    @ManyToMany(cascade = CascadeType.ALL)
+    @JoinTable(name = "book_user",
```

Comment:
I don't know if this is really necessary but it seems good for me :D

Reply:
I think it's not necessary 😄